reflect/src/main/java/dagger/reflect/Reflection.java

```diff
@@ -66,9 +68,19 @@
     }
   }
 
+  /**
+   * Finds scoping annotations that aren't {@link Reusable}.
+   *
+   * @param annotations The set of annotations to parse for scoping annotations.
+   * @return All annotations with Scope or an empty set if none are found.
+   */
   static Set<Annotation> findScopes(Annotation[] annotations) {
     Set<Annotation> scopes = null;
     for (Annotation annotation : annotations) {
+      // Reusable is ignored.
+      if (annotation.annotationType() == Reusable.class) {
```

Comment:
Where is the other check for this annotation? I think by moving it here we can delete the other one that already exists.